camel-idea-plugin/src/test/java/org/apache/camel/idea/annotator/CamelSimpleAnnotatorTestIT.java

```diff
@@ -61,14 +61,68 @@ public void testAnnotatorSimpleValidation() {
+       // myFixture.configureByText("AnnotatorTestData.java", getJavaWithPredicate());
+       // myFixture.checkHighlighting(false, false, true, true);
+    }
+
     private String getJavaWithSimple() {
         return "import org.apache.camel.builder.RouteBuilder;\n"
             + "public class MyRouteBuilder extends RouteBuilder {\n"
             + "        public void configure() throws Exception {\n"
             + "            from(\"netty-http:http://localhost/cdi?matchOnUriPrefix=true&nettySharedHttpServer=#httpServer\")\n"
             + "            .id(\"http-route-cdi\")\n"
             + "            .transform()\n"
-            + "            .simple(\"Response from Camel CDI on route<error descr=\"Unknown function: xrouteId\"> ${xrouteId} using thread: ${threadName}</error>\");"
+            + "            .simple(\"Response from Camel CDI on route<error descr=\"Unknown function: xrouteId\">${xrouteId}</error> using thread: ${threadName}\");"
+            + "        }\n"
+            + "    }";
+    }
+
+    private String getJavaOpenBracketWithSimple() {
+        return "import org.apache.camel.builder.RouteBuilder;\n"
+            + "public class MyRouteBuilder extends RouteBuilder {\n"
+            + "        public void configure() throws Exception {\n"
+            + "            from(\"netty-http:http://localhost/cdi?matchOnUriPrefix=true&nettySharedHttpServer=#httpServer\")\n"
+            + "            .id(\"http-route-cdi\")\n"
+            + "            .transform()\n"
+            + "            .simple(\"Response from Camel CDI on route${routeId} using thread: ${threadNam<error descr=\"expected symbol functionEnd but was eol\">e</error>\");"
+            + "        }\n"
+            + "    }";
+    }
+
+    private String getJavaMutlipleOpenBracketWithSimple() {
+        return "import org.apache.camel.builder.RouteBuilder;\n"
+            + "public class MyRouteBuilder extends RouteBuilder {\n"
+            + "        public void configure() throws Exception {\n"
+            + "            from(\"netty-http:http://localhost/cdi?matchOnUriPrefix=true&nettySharedHttpServer=#httpServer\")\n"
+            + "            .id(\"http-route-cdi\")\n"
+            + "            .transform()\n"
+            + "            .simple(\"Response from Camel CDI on route${routeId} using thread: ${threadNam<error descr=\"expected symbol functionEnd but was eol\">e</error>\");"
+            + "        }\n"
+            + "    }";
+    }
+
+    private String getJavaWithPredicate() {
+        return "import org.apache.camel.builder.RouteBuilder;\n"
+            + "public class MyRouteBuilder extends RouteBuilder {\n"
+            + "        public void configure() throws Exception {\n"
+            + "              from(\"direct:start\")\n"
+            + "                .loopDoWhile(simple(\"${body.length} <error descr=\"Unknown function: xrouteId\">=!=</error> 12\"))\n"
```

Comment:
Can you add a 2nd test where you do the variation of this as

    .filter().simple("xxx")

You genererally have 2 ways in java dsl, either as parameter

    .filter ( simple("xxx") )

Or as chained method call

    .filter . simple("xxx")

And that psi parser code is different depending on one other the other are used.